lib/Mojolicious/Static.pm

```diff
@@ -33,6 +33,9 @@ sub dispatch {
   $path = $stash->{path} ? $path->new($stash->{path}) : $path->clone;
   return undef unless my @parts = @{$path->canonicalize->parts};
 
+  # Don't serve when a part has embedded path separators on windows
+  if ( $^O eq 'MSWin32' or $^O eq 'cygwin' ) { m!\\! && return for @parts };
+
```

Comment:
Just a suggestion: Could you `use constant IS_WINDOWS => ($^O eq 'MSWin32' or $^O eq 'cygwin');` at the top of the file, and then use that 
```
if (IS_WINDOWS) { .... }
```
?
The if-statement should be removed at compile time so you'd get no speed degradation on Linux et al. I haven't tried this mind you.

Reply:
will do